.gitignore

```diff
@@ -123,3 +123,4 @@ docs
 .DS_Store
 todo
 package-lock.json
+/package-lock.json
```

Comment:
Why?

Reply:
My bad, it was intended to gitkeep package-lock, I'll add a patch commit to fix this.